content/content.css

```
@@ -210,6 +210,59 @@
 }
 
+/* Temporary highlight animations for saved highlights */
+.highlight-saver-temporary-saved {
+    animation: savedHighlightAppear 0.5s ease-out forwards;
+}
+
+@keyframes savedHighlightAppear {
+    0% {
+        opacity: 0;
+        transform: scale(0.8);
+        background-color: rgba(34, 197, 94, 0.8) !important;
```

Biome (2.1.2)

[error] Using !important within keyframes declaration is completely ignored in some browsers.

Consider removing useless !important declaration.

(lint/suspicious/noImportantInKeyframe)